terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/codelist/07_03_form.jsp

```diff
@@ -5,18 +5,24 @@
 <body>
   <h2>Codelist</h2>
   <br>
-  <form:form action="${pageContext.request.contextPath}/codelist" method="get" modelAttribute="codeListForm">
+  <form:form action="${pageContext.request.contextPath}/codelist"
+    method="get" modelAttribute="codeListForm">
     <table>
       <tr>
-        <td>Codelist which must be read with the use of multiple pattern: </td>
-        <td><form:select path="item1" items="${SAMPLE_CODELIST}" id="list1"/></td>
+        <td>Codelist which must be read with the use of multiple
+          pattern:</td>

```

Comment:
I captured multiple screen captures before and after formatting, compared the images, and confirmed that there was no difference other than date time, processing time, and `X-Track` .